config/tasks/train/train_resnet.yaml

```diff
@@ -1,8 +1,8 @@
 # @package _group_
-epochs: 1
+epochs: 20
 
-report_interval: 50
-val_interval: 100
+report_interval: 10
+val_interval: 10
```

Comment:
Should this be changed back, as per our discussion on slack?

Reply:
Yes